client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -64,9 +64,19 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 			return;
 		}
 
-		const { siteId } = this.props;
-
-		this.props.recordJpoEvent( 'calypso_jpo_business_address_submitted' );
+		const { settings, siteId } = this.props;
+
+		this.props.recordJpoEvent(
+			'calypso_jpo_business_address_submitted',
+			transform(
```

Comment:
I had to look this up, and consulted https://stackoverflow.com/questions/21536627/whats-the-difference-between-transform-and-reduce-in-lodash since I didn't quite get the difference between this and `reduce` from lodash docs 😅 
That said... why not use `reduce`? 😁

Reply:
I don't mind - `reduce` it is - https://github.com/Automattic/wp-calypso/pull/22385/commits/ab61e399baf039e24d17def7700c3e6f5541361f